delta/sourcing-psql/src/main/scala/ch/epfl/bluebrain/nexus/delta/sourcing/DeleteExpired.scala

```diff
@@ -32,18 +34,12 @@ object DeleteExpired {
   private val metadata: ProjectionMetadata = ProjectionMetadata("system", "delete-expired", None, None)
 
   /**
-    * Creates a [[DeleteExpired]] instance and schedules in the supervisor the deletion of expired ephemeral states
+    * Creates a [[PurgeProjection]] instance so that it can be scheduled in the supervisor
     */
-  def apply(supervisor: Supervisor, config: ProjectionConfig, xas: Transactors, clock: Clock[IO]): IO[DeleteExpired] = {
-    val deleteExpired = new DeleteExpired(xas, clock)
-
-    val stream = Stream
-      .awakeEvery[IO](config.deleteExpiredEvery)
-      .evalTap(_ => deleteExpired())
-      .drain
-
-    val deleteExpiredProjection =
-      CompiledProjection.fromStream(metadata, ExecutionStrategy.TransientSingleNode, _ => stream)
-    supervisor.run(deleteExpiredProjection).as(deleteExpired)
+  def apply(deleteExpiredEvery: FiniteDuration, xas: Transactors): PurgeProjection = {
```

Comment:
We just have to define a purge projection now without having to care about the supervisor part